frontend/src/components/PlayersHand/PlayersHand.tsx

```diff
@@ -0,0 +1,22 @@
+import { FC, ReactElement } from 'react';
+
+import Locations from '../../constants/locations';
+import Tile from '../../model/Tile';
+import TileContainer from '../TileContainer';
+
+const PlayersHand: FC = (): ReactElement => {
+  const tile = new Tile(
+    { bottom: Locations.FIELD, left: Locations.CITY, right: Locations.FIELD, top: Locations.CITY },
+    Locations.CITY,
+    false,
+  );
```

Comment:
Let's expect that tile will be passed as a prop (Gameboard will have information on the current tile in the hand